iamproperty/iamproperty.github.io
# Insight card

## Summary of Changes:
1. Add a new card component page to show each version of a card
2. Simplify the card deck component page
3. Add in new theme css colours that don't get overwritten on a dark backgrounds

## Rationale
Show the insight/news card

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/card

## Ticket(s)
DS-59

## Checklist

## Notes
